src/Extensions/S3Storage/OptionsDisplayStrings.resx

```diff
@@ -0,0 +1,164 @@
+    <value>The endpoint of S3 to connect to. This is optional in case of using AWS storage.</value>
+    <comment>The description provided for the Endpoint parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_AWSProfileName" xml:space="preserve">
+    <value>The AWS profile name to be used for login</value>
+    <comment>The description provided for the AWSProfileName parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_AWSProfilePath" xml:space="preserve">
+    <value>The AWS profile path, if profile details not stored in default path</value>
+    <comment>The description provided for the AWSProfilePath parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_RegionName" xml:space="preserve">
+    <value>The name of the S3 region</value>
+    <comment>The description provided for the RegionName parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_AccessKeyId" xml:space="preserve">
+    <value>The AWS AccessKeyId for IAM user to login</value>
+    <comment>The description provided for the AccessKeyId parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_SecretAccessKey" xml:space="preserve">
+    <value>The AWS SecretAccessKey associated AccessKeyId for IAM user to login</value>
+    <comment>The description provided for the SecretAccessKey parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_GeneratePreSignedUrl" xml:space="preserve">
+    <value>A boolean flag indicates if the return value of egress provider should be a pre-signed URL or only the bucket name and object id of uploaded entry.</value>
+    <comment>The description provided for the GeneratePresSignedUrl parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_PreSignedUrlExpiry" xml:space="preserve">
+    <value>The amount of time the generated pre-signed url will be accessible.</value>
+    <comment>The description provided for the PreSignedUrlExpiry parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_S3StorageEgressProviderOptions_ForcePathStyle" xml:space="preserve">
+    <value>The boolean flag set for AWS connection configuration ForcePathStyle option.</value>
+    <comment>The description provided for the ForcePathStyle parameter on S3StorageEgressProviderOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_CommonEgressProviderOptions_CopyBufferSize" xml:space="preserve">
```

Comment:
Note: I kept the name of this the same, even though it's no longer "common" - I can change this to be in-line with the others, but I left it as-is for now.